code/game/objects/items/food/martian.dm

```diff
@@ -520,7 +520,7 @@
 	crafting_complexity = FOOD_COMPLEXITY_3
 
 /obj/item/food/takoyaki/russian
-	name = "russian takoyaki"
+	name = "Russian takoyaki"
```

Comment:
Might need to be `\improper`

Reply:
You might be correct but isn't Russian a proper noun?

Reply:
Yup, makes sense. Thanks.